pauxy/walkers/multi_coherent.py

```diff
@@ -0,0 +1,384 @@
+import copy
+import numpy
+import scipy.linalg
+import random
+from pauxy.utils.linalg import sherman_morrison
+from pauxy.estimators.mixed import local_energy_multi_det_hh
+from pauxy.utils.misc import get_numeric_names
+from pauxy.walkers.stack import PropagatorStack, FieldConfig
+from pauxy.trial_wavefunction.harmonic_oscillator import HarmonicOscillator, HarmonicOscillatorMomentum
+
+class MultiCoherentWalker(object):
```

Comment:
Walkers should inherent from the base walker class. This was done recently, but avoids the need for everyone to have a get/set_buffer which can be dangerous.